dask/optimize.py

```diff
@@ -34,9 +34,11 @@ def cull(dsk, keys):
     return dict((k, v) for k, v in dsk.items() if k in seen)
 
 
-def fuse(dsk):
+def fuse(dsk, keys=None):
```

Comment:
I like `keys`.  I agree that in the context of `fuse` in isolation this is confusing.  In the context of optimizations in general this ends up being a common interface due to how `dask.get` works.